corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -227,7 +227,7 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
             var appId = Util.currentUrlToObject().appId;
             var data = this.options.model.get('data');
             var html = data.map(function (x) {
-                return DOMPurify.sanitize(md.render(x));
+                return DOMPurify.sanitize(md.renderInline(x));
```

Comment:
Interesting, what's the difference between the two functions?

Reply:
`render` has the [paragraph wraps ](https://github.com/markdown-it/markdown-it#simple)while `renderInline` does not. Some formatting issues that some users had was because the text had the paragraph wraps around them.

Reply:
I don't think this fixed the problem because I still see the issue on staging. What seems to be wrong is that the data for all columns (Contact ID, First Name, Last Name, etc.) is being concatenated together and repeated in every column. If you can test locally with multiple display properties in the case list, you will probably see the issue.

Reply:
Thanks @solleks I think I see the issue. Can you send me a link to a project so I can verify? I tested mine locally and on staging, but for some reason my setup doesn't allow for multiple fields.

Reply:
Does this link work? If you register a case there and then follow up, you can see the case list.

https://staging.commcarehq.org/a/ccqa2-2/cloudcare/apps/v2/#%7B%22appId%22%3A%222a0cfbc32c76484fb3dff4fcb7a20993%22%2C%22steps%22%3A%5B%221%22%5D%2C%22page%22%3Anull%2C%22search%22%3Anull%2C%22queryData%22%3A%7B%7D%2C%22forceManualAction%22%3Afalse%7D

If that whole link is no good, you can start here:

https://staging.commcarehq.org/a/ccqa2-2/cloudcare/apps/v2/#apps

and then click "Combobox Accessibility" and "Example cases".